README.md

```diff
@@ -104,6 +104,12 @@ counter.inc(); // Inc with 1
 counter.inc(10); // Inc with 10
 ```
 
+A counter can be reset manually.
```

Comment:
Maybe we could add what reset means, ie removing all label-value combinations and initialize to 0

Reply:
Good point. I'll extend the text.